doc/develop/getting_started/index.rst

```diff
@@ -186,13 +186,20 @@ The current minimum required version for the main dependencies are:
          .. code-block:: bat
 
             choco install cmake --installargs 'ADD_CMAKE_TO_PATH=System'
-            choco install ninja gperf python311 git dtc-msys2 wget 7zip
+            choco install ninja gperf python311 git dtc-msys2 wget 7zip strawberryperl
 
          .. warning::
 
             As of November 2024, Python 3.13 is not recommended for Zephyr development on Windows,
             as some required Python dependencies may be difficult to install.
 
+         .. note::
+            If the .pl extension has not yet been associated with an application,
+            then the first time a .pl file is run without specifying an
+            interpreter, Windows will ask what application will open Perl files.
+            Set the default app to Strawberry Perl. By default the executable is
+            installed at ``C:\Strawberry\perl\bin\perl.exe``.
+
```

Comment:
This note is important, but I think it should be moved to https://docs.zephyrproject.org/latest/contribute/guidelines.html#check-compliance-py? People checking out how to get started with Zephyr don't necessarily care about this... yet? (unless I missed someting and there is more occasions than running checkpatch locally where perl is needed)

Reply:
@kartben , great point!  Updated as you requested.  Thank you